load_balancer/src/load_balancer.py

```diff
@@ -0,0 +1,226 @@
+""" Webserver class definition. This webserver handles the incoming packets from both the
+    workload generator and web interface. This webserver  """
+
+""" TODO: 
+        - Change to be a DJango type web server (???) 
+        - Make more scalable
+        - !!! Add a method to turn off the server (I have no idea, tried signal handler - damon)
+        """ 
+
+import socket
+import sys
+import time
+import ast
+import queue
+from threading import Thread
+from threading import Lock
+import datetime as datetime
+
+
+cmd_print_lock = Lock()
+SERV_PORT       = 60000
+SERV_HOST_NAME  = '127.0.0.1'
+
+AUDIT_SERV_ADDY = '127.0.0.1'
+AUDIT_SERV_PORT = 50000
+DEFAULT_WEB_SERV_PORT = 65000
+
+NUM_FORWARD_SERVERS = 1
+
+# Table of open web servers to forward packet requests too. 
+#    - user IDs are key- hashed and modulos with NUM of servers
+#       this determines the server index to forward user request too
+#    - Audit server is always last array index. 
+forwardServers = []
+
+PACKET_SIZE     = 8192 
+
+class webServer():
```

Comment:
Is this supposed to be loadBalancer instead of webServer?

Reply:
Changed in next request